tasks/The  Beer Game.md

```diff
@@ -0,0 +1,86 @@
+![image](https://user-images.githubusercontent.com/78745728/130639721-4a907d9e-6d7a-4e2c-ab63-ebc82bb8ec3a.png)
+![image](https://user-images.githubusercontent.com/78745728/130639743-830c8c2d-2488-4d74-9eb6-2cf10b145071.png)
+![image](https://user-images.githubusercontent.com/78745728/130639764-db9d2747-dcbd-405f-89cc-1701095e3351.png)
+![image](https://user-images.githubusercontent.com/78745728/130639782-7f6cd759-ef1d-49cd-9bc2-9955c6f14e59.png)
+![image](https://user-images.githubusercontent.com/78745728/130639918-2d9a4891-6fe4-47da-bcd5-7cf33ba59af2.png)
+![image](https://user-images.githubusercontent.com/78745728/130639940-0f928ea1-d2e4-42a5-95e8-8d611c408c7e.png)
+![image](https://user-images.githubusercontent.com/78745728/130639963-27707d92-e67e-4404-b96f-42d7129639a1.png)
+![image](https://user-images.githubusercontent.com/78745728/130639986-4924b4b4-7c6f-4a55-918c-3a1fca1f52de.png)
+![image](https://user-images.githubusercontent.com/78745728/130640017-cf12c503-1be0-4858-baeb-cbf1c85cc7a0.png)
+![image](https://user-images.githubusercontent.com/78745728/130640041-f27c576b-f4e2-4c71-9cbe-a2773ea717e2.png)
+![image](https://user-images.githubusercontent.com/78745728/130640071-48700fe7-e99b-45d5-a16a-2a086bb3f8b6.png)
+![image](https://user-images.githubusercontent.com/78745728/130640090-96f74768-7fec-440a-bcb0-92c1063f7a7e.png)
+![image](https://user-images.githubusercontent.com/78745728/130640119-161c2b4c-2c43-4bd4-a4f4-d6aa6875ecee.png)
+![image](https://user-images.githubusercontent.com/78745728/130640139-2d9f9b4d-ecdd-4c89-b4e0-f6eca667704f.png)
+![image](https://user-images.githubusercontent.com/78745728/130640167-c103199f-cf6f-4ea9-aae4-ac400ea067b9.png)
+![image](https://user-images.githubusercontent.com/78745728/130640186-271da404-35ba-41f0-b0d5-098b34d94980.png)
+![image](https://user-images.githubusercontent.com/78745728/130640213-07efa0ad-2e8d-4e2d-93ea-01729341a65d.png)
+![image](https://user-images.githubusercontent.com/78745728/130640372-cedf9bdd-6665-4f05-990e-c7cbda25414d.png)
+![image](https://user-images.githubusercontent.com/78745728/130640394-c9bdaa8f-6ac4-4539-aea0-b7f524db50de.png)
+![image](https://user-images.githubusercontent.com/78745728/130640710-e7f7ddb5-f1c8-41ed-b8e3-51ac62680b36.png)
+![image](https://user-images.githubusercontent.com/78745728/130640736-6badb6e2-a719-4035-bddd-1870fca0b259.png)
+![image](https://user-images.githubusercontent.com/78745728/130640762-688f0da6-d05b-4bde-a2f6-1edb92d72f62.png)
+![image](https://user-images.githubusercontent.com/78745728/130640789-30556e25-5170-4d5c-ab92-b836787070a9.png)
+![image](https://user-images.githubusercontent.com/78745728/130640811-bb172bd1-051b-48bb-a050-9fac410bd169.png)
+![image](https://user-images.githubusercontent.com/78745728/130640838-0d19c371-ef02-4b23-8b74-dd8688286612.png)
+
+
+# Criteria
+## Performance calculation
+Costs for Team: goal is to have minimal costs per team 
```

Comment:
Maybe mention how these costs are accrued over time in the instructions, there may have been a screenshot pertaining the information but I did not see it. 

>"Holding costs are assessed at every station for on-hand inventories. Each station must satisfy the orders from its downstream player (or the customers) as much as possible. In case of a stockout, the excess is backlogged. A penalty cost is assessed only at the retail store for customer backorders. This reflects the supply chain’s desire to provide good customer service. (In contrast, the original beer game charges a penalty cost at each station." ( Fangruo Chen and Rungson Samroengraja, 4)